src/Admin/AbstractAdmin.php

```diff
@@ -2005,16 +2005,28 @@ public function isGranted($name, ?object $object = null): bool
 
     public function getUrlSafeIdentifier(object $model): string
     {
-        return $this->getModelManager()->getUrlSafeIdentifier($model);
+        try {
+            return $this->getModelManager()->getUrlSafeIdentifier($model);
+        } catch (\InvalidArgumentException $e) {
+            return '';
```

Comment:
This looks wrong. If there is a problem to get the identifier, you should see the error